index.html

```diff
@@ -507,7 +512,90 @@ <h3>StatusList2021Credential</h3>
+                cached for more than the <code>ttl</code> duration prior to the
+                start of verification operation on a <a>verifiable credential</a>.
+                Implementations that publish the status list SHOULD align
+                any protocol-specific caching information, such as the 
+                HTTP `Cache-Control` header, with the value in this field.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                credentialSubject.size
+              </td>
+              <td>
+                The <code>size</code> indicates the size of the status entry in bits.
+                <code>size</code> MAY be provided, but if <code>size</code> is not present
+                as a property of the <code>credentialStatus</code> then <code>size</code>
+                MUST be processed as `1`.  <code>size</code> MUST be an integer greater than zero.
+                if <code>size</code> is provided, and is greater than `1`, then the property
+                <code>credentialStatus.statusMessages</code> MUST be present, and the number of
+                status messages must equal the number of possible values.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                credentialSubject.statusMessages
+              </td>
+              <td>
+                The <code>statusMessages</code> property MUST be an array, and if present
+                the length of the array must equal the number of possible status states
+                indicated by <code>size</code>. <code>statusMessages</code> MAY be present if
+                <code>size</code> is `1`. <code>statusMessages</code> MUST be present if
+                <code>size</code> is greater than `1`.  If not present, the message value
+                associated with the bit value of <code>0</code> is "unset" and the bit
+                value of <code>1</code> is "set".
+                Elements in the <code>statusMessages</code> array if present, MUST contain at 
+                minimum two properties:
+                <ul>
+                  <li><code>status</code> being a string of the hex value of the status</li>
+                  <li><code>value</code> a string containing the associated message</li>
+                </ul>
+                Implementers MAY add additional values to objects in the <code>statusMessages</code>
+                array.
+                Implementers MAY add a boolean value to elements in the <code>statusMessages</code>
+                named <code>verify</code> which adds an instruction to verifiers where if the
+                value of <code>verify</code> is <code>true</code> the verifier SHOULD not
+                use the status to reject verification of the verifiable credential. If the
+                value of <code>verify</code> is <code>false</code>, then a verifier SHOULD
+                NOT verify the verifiable credential.
```

Comment:
```suggestion
                NOT use the status when verifying thenverifiable credential.
```

Reply:
fixing @andresuribe87 typo
```suggestion
                NOT use the status when verifying the verifiable credential.
```